### PR TITLE
chore(release): publish packages

### PR DESCRIPTION
**Pull request by spydon**
- ubuntu_service@0.3.2

### CODE REVIEW

**Review by d-loose — APPROVED** [review]

File: packages/ubuntu_service/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 0.3.2
```
Comment (d-loose) [inline packages/ubuntu_service/CHANGELOG.md:1]: Seems like the script doesn't pick up "chore" commits

Comment (spydon) [inline packages/ubuntu_service/CHANGELOG.md:1]:
```suggestion
## 0.3.2

 - **FIX**: Bump get_it to 7.6.9
```

Comment (spydon) [reply packages/ubuntu_service/CHANGELOG.md:1]: Yeah, that's intentional, I just forgot to add a line in the manual versioning step

Comment (d-loose) [reply packages/ubuntu_service/CHANGELOG.md:1]: Fractions of a second slower!

**Pull request merged**